components/controls/buttons/SettingsButton.tsx

```diff
@@ -9,10 +9,13 @@ import {
   useDisclosure,
   useToast,
 } from "@chakra-ui/react";
+import axios from "axios";
```

Comment:
No need to import this.

```suggestion
```

Reply:
Cool, yeah I'll remove this along with other unused imports in this file.